src/services/print.js

```diff
@@ -222,6 +234,8 @@ ngeo.Print.prototype.encodeTileLayer_ = function(arr, layer) {
   var source = layer.getSource();
   if (source instanceof ol.source.WMTS) {
     this.encodeTileWmtsLayer_(arr, layer);
+  } else if (source instanceof ol.source.TileWMS) {
+    this.encodeImageWmsLayer_(arr, layer);
```

Comment:
My plan was to have an `encodeTileWmsLayer_` function for tile layers with an `ol.source.TileWMS` source. And we can have an `encodeWmsLayer_` function if `encodeImageWmsLayer_` and `encodeTileWmsLayer_` have common code.